docs/items/tools.md

```diff
@@ -0,0 +1,246 @@
+    @Override
+    public int getDurabilityForType(ArmorItem.Type type) {
+        return switch (type) {
+            case HELMET -> 11 * 15;
+            case CHESTPLATE -> 16 * 15;
+            case LEGGINGS -> 15 * 15;
+            case BOOTS -> 13 * 15;
+        };
+    }
+
+    // Determines the defense value of this armor material, depending on what armor piece it is.
+    @Override
+    public int getDurabilityForType(ArmorItem.Type type) {
+        return switch (type) {
+            case HELMET -> 2;
+            case CHESTPLATE -> 4;
+            case LEGGINGS -> 6;
+            case BOOTS -> 2;
+        };
+    }
+
+    // Returns the toughness value of the armor.
+    // Only diamond and netherite have values greater than 0 here, so we just return 0.
+    @Override
+    public float getToughness() {
+        return 0;
+    }
+
+    // Returns the knockback resistance value of the armor.
+    // Only netherite has values greater than 0 here, so we just return 0.
+    @Override
+    public float getKnockbackResistance() {
+        return 0;
+    }
+
+    // Determines the enchantability of the tier. This represents how good the enchantments on this armor will be.
+    // Gold uses 25, we put copper slightly below that.
+    @Override
+    public int getEnchantmentValue(ArmorItem.Type type) {
+        return 20;
+    }
+
+    // Determines the sound played when equipping this armor.
+    @Override
+    public SoundEvent getEquipSound() {
+        return SoundEvents.ARMOR_EQUIP_GENERIC;
+    }
+
+    // Determines the repair item for this armor.
+    @Override
+    public Ingredient getRepairIngredient() {
+        return Ingredient.of(Tags.Items.INGOTS_COPPER);
+    }
+}
+```
+
+And then, we use that armor material in item registration.
+
+```java
+//ITEMS is a DeferredRegister<Item>
+public static final Supplier<ArmorItem> COPPER_HELMET = ITEMS.register("copper_helmet", () -> new ArmorItem(
+        // The armor material to use.
+        COPPER_ARMOR_MATERIAL,
+        // The armor type to use.
+        ArmorItem.Type.HELMET,
+        // The item properties. We don't need to set the durability here because ArmorItem handles that for us.
+        new Item.Properties()
+));
+public static final Supplier<ArmorItem> COPPER_CHESTPLATE = ITEMS.register("copper_chestplate", () -> new ArmorItem(...));
+public static final Supplier<ArmorItem> COPPER_LEGGINGS = ITEMS.register("copper_leggings", () -> new ArmorItem(...));
+public static final Supplier<ArmorItem> COPPER_BOOTS = ITEMS.register("copper_boots", () -> new ArmorItem(...));
+```
+
+Besides the usual resources, armors also need a worn armor texture that will be rendered over the player model when the armor is equipped. This texture must be located at `src/main/resources/assets/<mod_id>/textures/models/armor/<material>_layer_1.png` for the helmet, chestplate and boots textures, and in the same directory at `<material>_layer_2.png` for the leggings.
```

Comment:
Probably a mention to `#getArmorTexture` here in case for a custom location that doesn't follow the naming scheme.